server/src/main/java/org/elasticsearch/common/TimeBasedKOrderedUUIDGenerator.java

```diff
@@ -68,6 +68,6 @@ public String getBase64UUID() {
 
         assert buffer.position() == uuidBytes.length;
 
-        return BASE_64_NO_PADDING.encodeToString(uuidBytes);
+        return BASE_64_URL_ENCODER_NO_PADDING.encodeToString(uuidBytes);
```

Comment:
Let's just use `Strings.BASE_64_NO_PADDING_URL_ENCODER`, it's the same and used elsewhere too.